operations/companycontracts.md

```diff
@@ -83,3 +83,98 @@ Returns all contracts between the enterprise and other companies.
 | :-- | :-- | :-- | :-- |
 | `IncludeCancellationFeeInCommissionEstimate` | boolean | required | Cancellation fee will be considered when calculating the travel agency commission estimate. |
 | `SkipAutomaticSettlement` | boolean | required | Reservations from travel agencies will not be automatically charged. |
+
+## Add company contracts
+
+Adds a new company contracts.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/companyContracts/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "TravelAgencyContracts": [
+    {
+        "ServiceId": "c8f88563-dc60-47f3-aca3-af150065d951",
+        "CompanyId": "896e9313-477d-4306-9d37-af150065f4d6",
+        "Options": {
+            "IncludeCancellationFeeInCommissionEstimate": false,
+            "SkipAutomaticSettlement": false
+        },
+        "CommissionIncluded": null,
+        "Commission": null,
+        "ChannelManagerAbsoluteAdjustment": null,
+        "ChannelManagerRelativeAdjustment": null,
+        "AccountingCode": null,
+        "InvoiceDueInterval": null,
+        "ChannelManagerBusinessSegmentId": null,
+        "ContactPerson": null,
+        "ContactEmail": null,
+        "AdditionalContactInfo": null,
+        "Notes": null
+    }
+}
+```
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `TravelAgencyContracts` | array of [Travel agency contract parameters](#travel-agency-contract-parameters) | required | Information about travel agency contracts to be created. |
+
+#### Travel agency contract parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) the travel agency contract is created to. |
+| `CompanyId` | string | required | Unique identifier of the [Company](companies.md#company) the travel agency contract is issued to. |
+| `Options` | string | required | Options of the travel agency contract. |
```

Comment:
I assume this 'Options' property should not be here, because it appears twice, the second one below looks like the correct version.